src/tools/rust-analyzer/crates/parser/test_data/parser/ok/0035_weird_exprs.rs

```diff
@@ -64,8 +64,8 @@ fn notsure() {
 
 fn canttouchthis() -> usize {
     fn p() -> bool { true }
-    let _a = (assert!((true)) == (assert!(p())));
-    let _c = (assert!((p())) == ());
+    let _a = (assert!(true) == (assert!(p())));
+    let _c = (assert!(p()) == ());
```

Comment:
This will fail r-a's tests as the syntax tree is dumped in a neighboring file, probably best to not touch this

Reply:
Says it right there in the function name :stuck_out_tongue_winking_eye:.

(but seriously, it's fine to file this against RA if there's a decision to move forward with the change)

Reply:
reverted